jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -125,7 +126,8 @@ ecma_builtin_global_object_parse_int (ecma_value_t this_arg __attr_unused___, /*
   ecma_length_t end = str_size;
   for (ecma_length_t i = 0; i < end; i++)
   {
-    if (!(isspace (utf8_string_buff[i])))
+    if (!lit_char_is_white_space (utf8_string_buff[i])
+        && !lit_char_is_line_terminator (utf8_string_buff[i]))
```

Comment:
Is this correct? If I understand correctly, we just pass a byte, rather than an uint16_t character. I think we should rewrite this to use the UTF iterators, since there are line terminators and white spaces outside of the [0-127] byte range.